sri/templatetags/sri.py

```diff
@@ -41,11 +55,11 @@ def sri_static(path: str, algorithm: Optional[str] = None):
                 "crossorigin": "anonymous",
             }
         )
-    return sri_method(attrs, path, algorithm_type)
+    return sri_method(attrs, path, algorithm_type, **kwargs)
 
 
 @register.simple_tag
-def sri_integrity_static(path: str, algorithm: Optional[str] = None):
+def sri_integrity_static(path: str, algorithm: Optional[str] = None, **kwargs):
```

Comment:
Doesn't look like these `kwargs` are used here, and given what they're used for, probably shouldn't be either.